server.js

```diff
@@ -25,10 +26,32 @@ const handler = routes.getRequestHandler(app, ({ req, res, route, query }) => {
 
 app.prepare().then(() => {
   const server = express()
-  server.use(handler)
 
-  server.listen(PORT, err => {
+  // Authorization Cookie handle
+  server.use(cookieParser())
+  server.get('/token', (req, res) => {
+    if ('cookie' in req.headers) {
+      const cookieParts = req.headers.cookie.split('; ')
+      const authorization = cookieParts
+        .map(cookie => {
+          const cookieParts = cookie.split('=')
+          if (cookieParts.length && cookieParts[0] === 'csrftoken') {
+            return cookieParts[1]
+          }
+          return false
+        })
+        .filter(cookie => cookie !== false)
+      if (authorization.length) {
+        res.send(authorization[0])
+        return
+      }
+    }
+    res.status(204).send()
+  })
+
+  server.use(handler)
+  server.listen(5000, err => {
```

Comment:
We have a system environment that stores the PORT: `const PORT = process.env.PORT || 5000`

We should use this in the server listen and in the log.